app/report/_ReportResidencyFrTaxesFr.tsx

```diff
@@ -124,7 +124,9 @@ export const ReportResidencyFrTaxesFr: React.FunctionComponent<
           Benefit History) from Etrade.
         </div>
       </div>
-      {gainsAndLosses.length === 0 || fractionsFrIncome.length === 0 ? (
+      {gainsAndLosses.length === 0 ||
+      (gainsAndLosses.filter((e) => e.planType === "RS").length > 0 &&
```

Comment:
My understanding is that it applies to both `SO` and `RSU`s, so this should be something like this

```suggestion
      (gainsAndLosses.filter((e) => ["SO", "RS"].includes(e.planType)).length > 0 &&
```